CrowdFundingApplication/Views/Profile/Balance.cshtml

```diff
@@ -0,0 +1,53 @@
+﻿@model CrowdFundingApplication.Models.User
+
+@{
+    ViewData["Title"] = "Баланс";
+}
+
+<div style="height: 25px;"></div>
+<div class="mt-5">
+    <div class="row">
+        <div class="col">
+            <ul class="nav nav-tabs">
+                <li class="nav-item">
+                    <a class="nav-link" style="color: black;" asp-area="" asp-controller="Profile" asp-action="Index">Профиль</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" style="color: black;"asp-area="" asp-controller="Profile" asp-action="CreatedProjects">Созданные проекты</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link active" style="color: black;" aria-current="page" href="#">Управление балансом</a>
+                </li>
+            </ul>
+        </div>
+    </div>
+
+    <div class="row mt-1">
+        <div class="col">
+            <div class="card shadow-2-strong card-registration" style="border-radius: 5px;">
```

Comment:
click should not be in html

Reply:
I did not understand which exactly click?

Reply:
Has been edited